lib/active_storage_support/base64_attach.rb

```diff
@@ -0,0 +1,17 @@
+# Helper method to decode a base64 file and create a StringIO file
+module ActiveStorageSupport
```

Comment:
I think this code isn't part of this PR. Given this is a gem the PR should have `master` as the base branch instead of `development`